ImageGallery/src/org/sleuthkit/autopsy/imagegallery/gui/drawableviews/Bundle.properties-MERGED

```diff
@@ -1,4 +1,4 @@
-DrawableTileBase.externalViewerAction.text=Open in External Viewer
+DrawableTileBase.externalViewerAction.text=Open in External Viewer (Ctrl+E)
```

Comment:
does this shortcut actually work from imagegallery?

Reply:
It does not, I had attempted to remove those text changes from image gallery as a result of it not working in image gallery, (I added them before testing it there) looks like i missed some.

Reply:
That is too bad, It might not be that much work to make it work there also...

Reply:
ll take a look, I honestly only tried to get it working where we already had the bookmark shortcut working.

Reply:
Alright got it sorted out and working in image gallery

Reply:
awesome!